tools/javac_worker/src/build/please/compile/JavaCompiler.java

```diff
@@ -125,6 +132,7 @@ private BuildResponse reallyBuild(BuildRequest request) throws IOException {
             }
             opts.add(finder.joinFiles(':'));
             response.success = compiler.getTask(writer, fileManager, reporter, opts, null, compilationUnits).call();
+            fileManager.close();
```

Comment:
This could potentially be left open should an exception happen. We should declare the `fileManager` variable in a `try-with-resources` statement like so:
`try (StandardJavaFileManager fileManager = compiler.getStandardFileManager(reporter, null, null)) {`

Reply:
Thanks, I always forget those!